polkadot.json

```diff
@@ -83,37 +83,37 @@
       },
       {
         "id": 11,
-        "imageUrl": "https://i.imgur.com/YQNFrNk.png",
-        "title": "Kusama - Polkadot's wild cousin",
```

Comment:
I felt that 4 slides on Kusama was too much.

Instead I decided to focus on the key ecosystem parts of Polkadot:

- system chains, which we simply consider as part of Polkadot, and where polkadot uses itself to scale itself
- parachains, which extend the functionality of the network further
- and then kusama as a canary network

Reply:
sounds good!